common/changes/office-ui-fabric-react/groupheader-aria_2019-01-24-00-29.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Exposes props of selection check button in group header",
```

@natalieethell we can do suggestions in our comments 😄 
```suggestion
      "comment": "GroupHeader: exposes props of selection check button.",
```

---

Oh yeah I keep forgetting! Nice!